docs/source/develop/push_support.rst

```diff
@@ -17,6 +17,7 @@ Fermentrack currently supports three push targets:
 - **"Generic" Push Target** - Fermentrack's "native" push format - Pushes both specific gravity & temperature data
 - **Brewer's Friend** - Fermentrack's "native" push format - Pushes both specific gravity & temperature data
 - **Brewfather** - Fermentrack's "native" push format - Pushes both specific gravity & temperature data
+- **ThingSpeak** - Fermentrack's "native" push format - Pushes temperature data
```

Comment:
Oh wow. I actually need to rewrite all of these. I think I just copied & pasted, when I meant to actually provide some kind of explanation of what each was. Sorry about that!